### PR TITLE
React-Native: Add typings for ARTStatic Group + Shape fill

### PR DESCRIPTION
**Pull request by nikolasleblanc**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://medium.com/the-react-native-log/animated-charts-in-react-native-using-d3-and-art-21cd9ccf6c58>>

### CODE REVIEW

**Comment by iRoachie** [comment]
Can you provide a link to the rn source?

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
+//                 Nikolas LeBlanc <https://github.com/nikolasleblanc>
```
Comment (timwangdev) [inline types/react-native/index.d.ts]: Just wanting to make sure that you understand that by adding this line means you’ll be signing up as a maintainer and will get notifications for all issues/PRs.

**Review by timwangdev — CHANGES_REQUESTED** [review]
Please address @iRoachie 's comment.

**Comment by RyanCavanaugh** [comment]
@nikolasleblanc Please address comments from the code reviewers.

**Comment by alloy** [comment]
Bump.

**Comment by alloy** [comment]
@nikolasleblanc I’m going to close this PR for now. Please do re-open when/if you fix these merge conflicts.

**Pull request closed without merging**